doc/install/install.md

```diff
@@ -1,32 +1,120 @@
+
+## Release binary
+
 Check out the latest [releases on github](https://github.com/operator-framework/operator-lifecycle-manager/releases) for release-specific install instructions.
 
-## Manual installation 
+## Minikube 
+
+This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml](local-values.yaml) to build localized deployment resources for OLM.
+
+```bash
+$ make run-local
+```
+
+You can verify that the OLM components have been successfully deployed by running `kubectl -n local get deployments`
+
+**NOTE** It is recommended for development pruposed and will use the source locally
+
+## Minishift
+
+Following the steps. 
+
+* Run the following command to allow your Minishift work wth operators. 
 
-Installing the CRDs first gives them a chance to register before installing the rest, which requires the CRDs exist.
 ```bash
-kubectl create -f deploy/upstream/quickstart/crds.yaml
-kubectl create -f deploy/upstream/quickstart/olm.yaml
+# add addon to allow work with operators
+$ minishift addon enable admin-user
+```
+
+* Re-started your Minishift
+
+```bash
+# minishift stop
+$ minishift start
+```
+* Run the following command to install. 
+
+```bash
+$ make install-ocp
 ```
 
 ## OpenShift
 
-OLM is installed by default in OpenShift 4.0 and above.
+**IMPORTANT:** OLM is installed by default in OpenShift 4.0 and above.
 
-## Run locally with minikube
+### By OCP 3.11 installation scripts
 
-This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml](local-values.yaml) to build localized deployment resources for OLM.
+Following the steps required to enable the OLM for an OSD OpenShift installation.
+
+* Change the variables in the inventory file as follows.
+
+```yaml
+# Deploy Operator Lifecycle Manager Tech Preview 
+# See more about it here: https://docs.openshift.com/container-platform/3.11/install/configuring_inventory_file.html#configuring-the-operator-lifecycle-manager
 
+openshift_enable_olm=true
 ```
-make run-local
+
+```yaml
+# For Operator Framework Images 
+# Note the "'test_image':'mongodb/enterprise-operator:0.3.2'"
+openshift_additional_registry_credentials=[{'host':'registry.connect.redhat.com','user':'REPLACE_ME','password':'REPLACE_ME','test_image':'mongodb/enterprise-operator:0.3.2'}]
 ```
+For further information see the [documentation](https://docs.openshift.com/container-platform/3.11/install_config/installing-operator-framework.html)
 
-You can verify that the OLM components have been successfully deployed by running `kubectl -n local get deployments`
+* Allow permission to the users
+
+On 3.11, it is a tech preview then just users with cluster admin permissions are able to use it. Following the command to give access to the users.
+
+```shell
+$ oc adm policy add-cluster-role-to-user cluster-admin <user>
+```
+
+### OCP 3.11 
```

Comment:
Officially, OLM (master/latest/etc) is not supported in 3.11, just the particular version that shipped as a tech preview. We shouldn't encourage users to install this way.

Reply:
The problem is that we still using minishift and OCP. Wdyt over add some info as (unsupported) and keep the steps?

Reply:
Solved.